docs/templates/README.md

```diff
@@ -0,0 +1,34 @@
+# Templates
+
+Find all provided documentation templates in this folder. Please note that the _italic text 
+and sentences_ should be removed. Feel free to add additional sections and subsections, however, make sure
+that at least the sections of the templates marked as "mandatory" are filled.
+
+## Decision Records
+
+[Link](decision-record.md) to template.
+
+Each decision record should be put in an approriate [folder](../developer/decision-records) 
```

Comment:
You mean that under `developer/decision-records` should a folder be created, right? Then I would write it like this to make it more clearer:
```suggestion
Each decision record should be put in an appropriate folder under this [folder](../developer/decision-records) 
```